language/move-compiler/tests/move_check/parser/num_literal_underscore_trailing.move

```diff
@@ -0,0 +1,6 @@
+module 0x42::M {
+    fun t() {
+        // Trailing underscore not allowed
+        let _ = 0u8_;
```

Comment:
It is though right? Trailing underscore just isn't allowed after a type suffix. Like `0_` should be valid?

Reply:
Yes thats valid and tested in the positive tests